usermods/bluetooth_serial/bluetooth_serial.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+#include "wled.h"
+#include "BluetoothSerial.h"
+#include "esp_bt_device.h"
+
+BluetoothSerial SerialBT;
+
+class BluetoothSerialUsermod : public Usermod {
+ 
+  public:
+
+
+    void setup() {
+
+      // Tell WLED to defer all WiFi Connectivity
+      deferConnections = true;
+      
+      // Begin Bluetooth device with same name as server description
+      SerialBT.begin(serverDescription);
+
+      Serial.println("Bluetooth Serial Usermod setup complete");
```

Comment:
Use DBUG_PRINTLN() and use flash strings F()

Reply:
Is your thinking here to replace all serial out with DEBUG equivalent or just this message?
Getting things like the bluetooth device name and hardware address can be useful, but I can see how they may belong as a debug message instead.

Reply:
Avoid using Serial.print() for printing debug messages. Use it only if you intend output to Serial interface.
Debug can go to networked recipient.